tlsdefaults.go

```diff
@@ -11,9 +11,7 @@ var (
 	preferredCipherSuites = []uint16{
 		tls.TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA,
 		tls.TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA,
-		tls.TLS_ECDHE_RSA_WITH_RC4_128_SHA,
 		tls.TLS_ECDHE_RSA_WITH_3DES_EDE_CBC_SHA,
-		tls.TLS_RSA_WITH_RC4_128_SHA,
```

Comment:
Yeah consulted [Go code](https://golang.org/src/crypto/tls/cipher_suites.go#L75), the default cipher suites are the 16 supported suites minus the 4 RC4 ones (with `suiteDefaultOff` flag). Seems good enough.